plugins/parcelport/verbs/unordered_map.hpp

```diff
@@ -0,0 +1,354 @@
+////////////////////////////////////////////////////////////////////////////////
+//  Copyright (c) 2016 John Biddiscombe
+//
+//  Distributed under the Boost Software License, Version 1.0. (See accompanying
+//  file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+////////////////////////////////////////////////////////////////////////////////
+
```

Comment:
As per comment above, the plan is to move this to hpx::concurrent once things settle down. this unordered map code is now well tested and has proven to be very useful with the readers/writers lock guarding it.